guides/source/action_mailbox_basics.md

```diff
@@ -324,9 +324,10 @@ end
 
 ## Incineration of InboundEmails
 
-By default, an InboundEmail that has been successfully processed will be
-incinerated after 30 days. This ensures you're not holding on to people's data
-willy-nilly after they may have canceled their accounts or deleted their
+By default, an InboundEmail that has been processed will be
```

Comment:
The "processing" here means `processed?` status- https://github.com/rails/rails/blob/6f6c211f4781e7bda63e5f47089f9d2612dc0a52/actionmailbox/app/models/action_mailbox/inbound_email.rb#L44-L44

Think it is correct as is.

Reply:
The `processed?` included the fails status. It was a little bit confusing to me. Even if the job fails with exceptional error, incineration happens. I thought this did not match the current doc.

Reply:
But I can get what you mean, so this PR should be improved.

Reply:
I fixed it.
https://github.com/rails/rails/pull/49743/commits/9153b8f9c2f812d1d42466a65de74f3e88d3026d